pkg/bot/mattermost_fmt.go

```diff
@@ -0,0 +1,97 @@
+package bot
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/kubeshop/botkube/pkg/events"
+	format2 "github.com/kubeshop/botkube/pkg/format"
+)
+
+func (b *Mattermost) longNotification(event events.Event) []*model.SlackAttachmentField {
+	fields := []*model.SlackAttachmentField{
+		{
+			Title: "Kind",
+			Value: event.Kind,
+			Short: true,
+		},
+		{
+			Title: "Name",
+			Value: event.Name,
+			Short: true,
+		},
+	}
+
+	if event.Namespace != "" {
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Namespace",
+			Value: event.Namespace,
+			Short: true,
+		})
+	}
+
+	if event.Reason != "" {
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Reason",
+			Value: event.Reason,
+			Short: true,
+		})
+	}
+
+	if len(event.Messages) > 0 {
+		message := ""
+		for _, m := range event.Messages {
+			message += fmt.Sprintf("%s\n", m)
+		}
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Message",
+			Value: message,
+		})
+	}
+
+	if event.Action != "" {
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Action",
+			Value: event.Action,
+		})
+	}
+
+	if len(event.Recommendations) > 0 {
+		rec := ""
+		for _, r := range event.Recommendations {
+			rec += fmt.Sprintf("%s\n", r)
+		}
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Recommendations",
+			Value: rec,
+		})
+	}
+
+	if len(event.Warnings) > 0 {
+		warn := ""
+		for _, w := range event.Warnings {
+			warn += fmt.Sprintf("%s\n", w)
+		}
+
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: "Warnings",
+			Value: warn,
+		})
+	}
+
+	// Add clusterName in the message
+	fields = append(fields, &model.SlackAttachmentField{
```

Comment:
if it's always here, please add it in the initial var (line 13)

Reply:
I don't think I can, as the fields order matter I think 🤔